### PR TITLE
Add new drink Mango Lassi

### LINKED ISSUE #20: Add a drink! 🍸
Hacktoberfest starts today! Add your drink recipes by forking and submitting a pull request.

Check back for updates for upcoming challenges. I'll be adding feature requests and bug fix requests for varying difficulty levels.

Also, be sure to check out the [Contributing Guidelines](https://github.com/alfg/opendrinks/blob/master/CONTRIBUTING.md) before submitting a pull request. :)

Happy hacking!

### PR DESCRIPTION
**Pull request by parammittal16**
Fixes #20 @alfg plz review

### CODE REVIEW

File: src/recipes/mango-lassi.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Mango Lassi",
+    "description": "This recipe comes from the cookbook author Madhur Jaffrey, who said, 'Mangoes are supposed to be good for the eyes.'",
+    "github": "prammittal16",
```
Comment (alfg) [inline src/recipes/mango-lassi.json]: Typo here? This comes up as 404.

**Comment by parammittal16** [comment]
But it is showing fine on both deploy preview and locally
Capture

File: src/recipes/mango-lassi.json (hunk 1 above)
Comment (parammittal16) [reply src/recipes/mango-lassi.json]: Done @alfg

**Comment by alfg** [comment]
Thanks!

**Pull request merged**